src/main/scala/ai/privado/languageEngine/java/language/package.scala

```diff
@@ -36,12 +36,12 @@ package object language {
 
   implicit class StepsForProperty(val trav: Traversal[JavaProperty]) extends AnyVal {
 
-    def usedAt: Traversal[CfgNode] = trav.out(EdgeTypes.IS_USED_AT).cast[CfgNode]
+    def usedAt: Traversal[AstNode] = trav.out(EdgeTypes.IS_USED_AT).cast[AstNode]
```

Comment:
We don't need to change `CfgNode` to `AstNode`.

Reply:
Done